include/invert_quda.h

```diff
@@ -1117,22 +1098,32 @@ namespace quda {
 
   class MR : public Solver {
 
-  private:
```

Comment:
any reason why `private:` is deleted here?

Reply:
Simply that class members default to `private` so this is redundant.

Reply:
Sure. But then we are not really consistent as it's the only `private` that was deleted hence my ask.

Reply:
Restored for consistency: a285361b8900d516da4af9bb79237bcd6adf1c58